apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -96,7 +98,7 @@ function formatResponse(data) {
 }
 
 export default async function fetchDatasets(organization, query = {}) {
```

Comment:
```suggestion
export default async function fetchDatasets(organization, pathname, query = {}) {
```

Since `pathname` is now required, make it a standalone arg.